charts/calling-test/values.yaml

```diff
@@ -0,0 +1,24 @@
+replicaCount: 1
+image:
+  # note: the docker image tag is configured as 'appVersion' in Chart.yaml
+  repository: quay.io/wire/avs-nwtesttool
+  pullPolicy: IfNotPresent
+
+envVars:
+  # note: this should be overridden in every deployment
+  BACKEND_HTTPS_URL: https://nginz-https.example.com
+
+nameOverride: "calling-test"
```

Comment:
If someone overrides this name to something else, than the routing on the `nginz` level will no longer work, right? Maybe add a note along the lines of "Do not change this name unless you know what you are doing, routing over nginz will not work"?